package.json

```diff
@@ -7,6 +7,7 @@
     "build:truffle-contract-schema": "cd packages/truffle-contract-schema && npm run build",
     "lerna:bootstrap": "lerna bootstrap",
     "bootstrap": "./scripts/bootstrap.sh",
+    "postinstall": "./scripts/downloadSolc.sh",
```

Comment:
Can we just inline the script here? I'm worried about non-UNIX operating systems where that file might not be compatible.

Should be fine if the entirety of the `postinstall` is just `truffle compile --download-solc 0.5.4`

If it turns into more than one line later, where a shell script becomes more warranted, we'll have to look into cross-platform solutions then.